pkg/network/event_common.go

```diff
@@ -277,6 +277,12 @@ type IPTranslation struct {
 	ReplDstPort uint16
 }
 
+// connectionTwoTuple represents one end of a connection
+type connectionTwoTuple struct {
```

Comment:
consider renaming to `connectionPeer`

Reply:
Not sure why peer is relevant in this context.
One possible name for the connectionTwoTuple struct could be `connectionEndpoint`. This name better captures the fact that the struct represents one endpoint of a connection.
Let me know what's your opinion about it.